docs/dev-guide/adding-grid-support/adding-grid-support-SE-grid-overview.md

```diff
@@ -0,0 +1,23 @@
+# Atmosphere Grid Overview
```

Comment:
Can this page be moved to eam/docs?

Reply:
Yea, I guess so... but it feels more appropriate to have it in the general grid support guide since it sometimes applies to the land. I feel like things will get chaotic if we start moving each page into the component documentation based on which component is most represented in each page.

Reply:
Sure, I'll move it.

Reply:
what do you think of putting it in `components/eam/docs`?

Reply:
This page would be in components/eam/docs/tech-guide.  The figures go in components/eam/docs/figures

Reply:
ok, did some more reorganization.